tests/Frontend/e2e/UserProfile.cy.js

```
@@ -107,4 +107,20 @@ describe("User Profile", function () {
+      userData.data.external_image = true;
+      cy.intercept("GET", "api/v1/users/1", userData).as("userRequest");
+    });
+    
+    cy.visit("/profile");
+    cy.wait("@userRequest");
+
+    cy.get('[data-test="profile-image-field"]').should('be.visible');
+    cy.get('[data-test="upload-file-button"]').should('not.exist');
+    cy.get('[data-test="delete-image-button"]').should('not.exist');
+    
+  });
```

ESLint

[error] Do not pass arrow functions to it()

(mocha/no-mocha-arrows)